packages/compass-connections/src/stores/connections-store.ts

```diff
@@ -361,7 +362,7 @@ export function useConnections({
       // not resolved.
       connectionsManager.cancelAllConnectionAttempts();
     };
-  }, [getAutoConnectInfo, persistOIDCTokens]);
+  }, [persistOIDCTokens]);
```

Comment:
I'm maybe missing something in the diff here, but is `persistOIDCTokens` being used in this hook at all? Why are we depending on this for the initial autoconnect?

Reply:
You're right. We don't need, sorry for the oversight.